app/MonsterConverterTfsXml/TfsXmlConverter.cs

```diff
@@ -518,7 +518,6 @@ private XElement SpellGenericToTfsXml(Spell spell, string elementName)
                 {
                     ability.Add(new XAttribute("name", spell.Name));
                     ability.Add(new XAttribute("interval", spell.Interval));
-                    ability.Add(new XAttribute("chance", Math.Round(spell.Chance * 100)));
 
```

Comment:
Chance is a valid attribute for melee attacks on TFS enginer. For cipbia monsters its always 100% chance. There could be users of this tool that set the melee chance for their custom monsters. 

It looks like chance defaults to 100% for TFS, instead of removing it completely lets put it behind a condition.

if (spell.Chance < 1) {
ability.Add(new XAttribute("chance", Math.Round(spell.Chance * 100)));
}